### PR TITLE
Standardize usage of the 'Maslow' string

### PR DESCRIPTION
**Pull request by aersida**
Standardizes (standardises) most usages of "Maslow" in various strings.

Also does some clean up of the related code trying to do things the JavaScript way (ish).

### CODE REVIEW

**Comment by BarbourSmith** [comment]
I shouldn't be merging this, right?

**Review by BarbourSmith — APPROVED** [review]
Excellent clean up!

**Comment by aersida** [comment]
> I shouldn't be merging this, right?

You should now...

File: www/js/maslow.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        return true;
+    }
+
+    return false;
+}
+
+/** Perform maslow specific-ish error message handling */
+const maslowErrorMsgHandling = (msg) => {
+    if (!msg.startsWith("error:")) {
+        // Nothing to see here - move along
```
Comment (BarbourSmith) [inline www/js/maslow.js:42]: At first I thought this comment was for the reader, as in "don't worry about this code, just keep reading 😛

File: www/js/modaldlg.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ var listmodal = [];
 
 function setactiveModal(html_template, closefunc) {
     if (typeof id(html_template) === 'undefined') {
-        console.log("Error: no " + html_template);
+        console.error(`Error: no ${html_template}`);
```
Comment (BarbourSmith) [inline www/js/modaldlg.js:7]: These are a great change

File: www/js/settings.js [hunk 3]
```diff
@@ -6,7 +9,7 @@ var current_setting_filter = 'nvs'
 var setup_is_done = false
 var do_not_build_settings = false
 const CONFIG_TOOLTIPS = {
-  Maslow_vertical: "If the maslow is oriented horizontally, set this to false",
+  Maslow_vertical: `If the ${M} is oriented horizontally, set this to false`,
```
Comment (BarbourSmith) [inline www/js/settings.js:12]: What is the benefit here if we aren't trying to save memory on this side?

File: www/js/tests/maslow.test.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
+import { beforeEach, describe, expect, test } from "bun:test";
```
Comment (BarbourSmith) [inline www/js/tests/maslow.test.ts:1]: Awesome 😀

**Review by BarbourSmith — APPROVED** [review]
Phenomenal!

**Comment by BarbourSmith** [comment]
Was this change intentional? 

image

It seems like the gulp build process is broken. I know that the long term plan is to get rid of it entirely, but ATM it's what we've got 😛

**Pull request merged**